lib/happo/public/happo-styles.css

```diff
@@ -1,4 +1,11 @@
 body {
   background-color: #f0f0f0;
   font-family: helvetica, arial;
+  background: repeating-linear-gradient(
+    -45deg,
+    #f0f0f0,
+    #f0f0f0 10px,
+    #f4f4f4 10px,
+    #f4f4f4 20px
+  );
```

Comment:
I'll check out what you pushed soon. I tried a checkerboard first, but it looked like a grid and things weren't aligning well on top of it (headings mostly). .